### PR TITLE
0.2.0 docs

### PR DESCRIPTION
**Pull request by cwurtz**

### CODE REVIEW

**Comment by tedmiston** [comment]
@cwurtz Overall LGTM.  I added a few comments and made some formatting changes.

File: docs/pages/guides/google-cloud.md [hunk 1]
```diff
@@ -29,6 +29,24 @@ Private clusters is currently a beta feature on Google Kubernetes Engine (GKE) t
...
+export YOUR_EMAIL_ADDRESS=you@yourdomain.com
+export NAMESPACE=astronomer
+```
+
+If you aren't sure about the `PROJECT_ID`, you can list your GCP projects with
+```bash
+gcloud projects list
+```
+
+Create a static IP address with the following command:
```
[the remark this answers is not in the document]
Comment (cwurtz) [reply docs/pages/guides/google-cloud.md]: Ugh, yeah was a merge conflict, guess I botched it

File: docs/pages/guides/google-cloud.md [hunk 2]
```diff
@@ -77,46 +99,35 @@ The Astronomer Platform requires several secrets to be in place before installat
...
 
-WARNING: The examples here use the `--from-literal` option to demonstrate the format of the values, but you can just as easily create the secrets from txt files, using the `--from-file` option. This would look something like this: `kubectl create secret generic airflow-metadata --from-file connection=airflow-metadata.txt`, where the content of the `txt` file is the connection string. If you do use the `--from-literal` form, the secrets will most likely be hanging around in your shell's history, which could be a security concern.
-
-The following commands are using the default secret names specified in the root `values.yaml`. If you change the names, make sure you update your `config.yaml` file. These values are located under `airflow.data`. Also, don't forget to switch your `kubectl` default context to the namespace you created earlier, or specify the namespace with the `--namespace` flag.
-
-<!-- markdownlint-disable MD036 -->
-*Note: Pay careful attention to the 3 different styles of PostgreSQL URI schemes required below (`postgresql://...`, `db+postgresql://...`, `postgres://...`).*
-<!-- markdownlint-enable MD036 -->
```
[the remark this answers is not in the document]
Comment (cwurtz) [reply docs/pages/guides/google-cloud.md]: Yeah, no longer needed.  Going forward, Houston will handle the creation of the secrets that had inconsistent URI's. 

In v0.2.0, we just need a secret for houston, and then one for airflow.  Using the airflow connection we'll create databases/schemas for airflow, celery, and grafana, and the secrets for those connections.  The appropriate URI will be generated depending on the application meant to consume it.

File: docs/pages/guides/google-cloud.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (cwurtz) [reply docs/pages/guides/google-cloud.md]: Removed it, not sure why that was there to begin with

**Pull request merged**